ministryofjustice/correspondence_tool_staff
# Change button colour for find address

## Description
Updates button colour to be in line with colour meanings of design system.

## Self-review checklist



### Screenshots
Before:
<img width="601" alt="Screenshot 2021-10-13 at 16 46 35" src="https://user-images.githubusercontent.com/13377553/137168287-b1b0b0ae-350c-4cd6-aadc-bc543e4c09e9.png">


After:
<img width="628" alt="Screenshot 2021-10-13 at 16 45 55" src="https://user-images.githubusercontent.com/13377553/137168168-37f8cf4d-20c8-4df7-b25d-a8daae9ecfbe.png">


### Related JIRA tickets


### Deployment


### Manual testing instructions
- start a new case
- search for an address
- make sure everything works okay (i.e. search, paste, re-paste, form submission)
